CMakeLists.txt

```diff
@@ -201,6 +201,15 @@ if(CMAKE_SYSTEM_PROCESSOR MATCHES "ppc64le")
   endif(HAS_ALTIVEC)
 endif(CMAKE_SYSTEM_PROCESSOR MATCHES "ppc64le")
 
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64|AARCH64")
+        CHECK_C_COMPILER_FLAG("-march=armv8-a+crc" HAS_ARMV8_CRC)
+  if(HAS_ARMV8_CRC)
+    message(STATUS " HAS_ARMV8_CRC yes")
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -march=armv8-a+crc -Wno-unused-function")
```

Comment:
is there any particular reason for allowing unused functions?

Reply:
Really thanks for your comments.
when 'HAS_ARMV8_CRC' is enabled, 'rocksdb::crc32c::isPCLMULQDQ() / isSSE42() ' will not be used but have been actually defined. So it's needed to suppress the warning here.